### DISCUSSION TITLE
feature: have vi compatible keymap

### DISCUSSION THREAD

**Discussion by muellert** [LazyVim/LazyVim#4674] [Ideas]
### Did you check the docs?

- [x] I have read all the LazyVim docs
- [x] This is not a request for a new or existing extra (see above)

### Is your feature request related to a problem? Please describe.

I'd like to have a plain 'vi' or 'vim' compatible keymap, or a more exhaustive documentation about how to make one.

### Describe the solution you'd like

I'd like to be able to switch on a "vim compatible" keymap that would make most common operations use the same keys or key combinations as in plain vim, or a more exhaustive documentation about how to make one.

### Describe alternatives you've considered

n/a

### Additional context

Background: I have one computer on which I can run LazyVim, but a number of computers where I can only run 'vi' or plain 'vim', and where installing extensions is not feasible or possible (servers, other people's computers etc.). Having the "same" keymap everywhere would significantly reduce friction, as opposed to the current situation, where using LazyVim and using plain vim is practically like using two entirely unrelated editors. As is, using muscle memory from other vi or vim installations is rarely possible.

**Comment by dpetka2001** [comment]
Vim and Neovim are different editors. Vim requires a `.vimrc` to be configured in Vimscript. While Neovim does support configuring in Vimscript (and even loading from a `.vimrc` file if I'm not mistaken?), I don't think that's in scope with this project's goal. LazyVim is a Neovim setup not a Vim one.

**Comment by muellert** [comment]
I am aware that LazyVim is an add-on to NeoVim, but I hoped it might be possible to make the "vi" family of editors more compatible with each other. Not having to re-learn yet another editor for once would be nice.
